cfme/tests/cloud/test_providers.py

```diff
@@ -377,6 +376,9 @@ def test_select_key_pair_none_while_provisioning(appliance, request, has_no_clou
         4. Select None in Guest Access Key Pair
         5. None should be selected
     """
+    if 'govcloud' in provider.data.tags:
```

Comment:
**question** why do we turn the uncollects into skips?

Reply:
DataProvider comes in uncollect as provider. DataProvider doesn't provide any provider data.
So, it was agreed to turn complex uncollects into skips.